apiserver/common/relationunitswatcher.go

```diff
@@ -19,9 +19,11 @@ type RelationUnitsWatcher interface {
 	watcher.Watcher[params.RelationUnitsChange]
 
 	// Stop is needed to implement facade.Resource.
+	// Deprecated: use Kill and Wait instead.
 	Stop() error
 
 	// Err implements watcher.Errer.
+	// Deprecated: use Kill and Wait instead.
 	Err() error
```

Comment:
Leave these for now, there are a lot of tests that depend on `watcher.EnsureErr()` and we need to undo that. Just not in this PR.